reana_client/cli/utils.py

```diff
@@ -127,7 +127,34 @@ def parse_parameters(_filter):
         )
 
 
-def filter_data(parsed_filters, headers, tablib_data):
+def parse_filter_parameters(filters):
+    """Return parsed filter parameters."""
+    try:
+        status_filters = []
+        search_filters = None
+        filters = " ".join(filters).split(",")
+        for item in filters:
+            if "=" in item:
+                filter_name = item.split("=")[0].lower()
+                filter_value = item.split("=")[1]
+                if filter_name == "status":
+                    status_filters.append(filter_value)
```

Comment:
Maybe we can check here that the provided status is a valid one? Otherwise:

```console
$ reana-client list --filter status=foo
Workflow list could not be retrieved: 
Malformed request.
```

Do you know if we have somewhere in r-commons all the workflow statuses to do this check? cc/ @diegodelemos @tiborsimko

Another option would be returning a more descriptive exception from r-server when the provided status is invalid.

This doesn't necessarily need to be addressed now. Maybe we can create an issue for it.

Reply:
@mvidalgarcia  What are the different status possible, I can create a mapping for them?

Reply:
Validation would be very good to do :+1: The list of statuses is living in `reana-db` enum, see `class  WorkflowStatus` there. But we cannot use `r-db` here in `r-client`... So we'd either have to pull it out into to `r-commons`, or simply hard-code here for the MVP... That should do, since they are not changing much as a function of time. So we could simply enrich `r-client/config.py` with the status list for now...

Reply:
Done 👍